omegaconf/grammar_visitor.py

```diff
@@ -54,7 +54,7 @@ def __init__(
             three keyword arguments: `name` (str, the name of the resolver),
             `inputs` (tuple, the inputs to the resolver), and `inputs_str` (tuple,
             the string representation of the inputs to the resolver -- for backward
-            compatibility with resolvers registered with `args_as_strings=True`).
+            compatibility with resolvers registered with `legacy_register_resolver()`).
```

Comment:
If we don't have one, can you file an issue to clean up legacy_register_resolver after 2.1?

Reply:
> If we don't have one, can you file an issue to clean up legacy_register_resolver after 2.1?

Done in #440 (can you please assign it the 2.2 milestone? I don't have permissions)

Reply:
did it before I even saw your comment :)